cpp/include/cuvs/neighbors/ivf_flat.hpp

```diff
@@ -436,6 +460,10 @@ void build(raft::resources const& handle,
 /**
  * @brief Build the index from the dataset for efficient search.
  *
+ * Note, if dataset is in host memory and index_params.add_data_on_build = true, user could set a
+ * stream pool in the input raft::resource with (at least) one stream to enable kernel/copy
+ * overlapping.
+ *
  * Usage example:
  * @code{.cpp}
  *   using namespace cuvs::neighbors;
```

Comment:
I agree with Corey's last comment: it would be great to show here how to create the stream pool
```suggestion
 *   using namespace cuvs::neighbors;
 *   raft::resources handle;
 *   // Add an extra stream to overlap dataset copies with computation during index building.
 *   raft::resource::set_cuda_stream_pool(handle, std::make_shared<rmm::cuda_stream_pool>(1));
```

Reply:
Thanks for the suggestion. I added the examples